pages/content/amp-dev/documentation/guides-and-tutorials/start/create/actions_events.md

```diff
@@ -0,0 +1,42 @@
+---
+$title: Actions and events
+$order: 5
+description: 'AMP embraces interactive user experiences.'
+author: crystalonscript
+---
+
+AMP embraces interactive user experiences. But, for performance and user experience guarantees, it’s just done a bit differently than non-AMP pages.
+
+AMP uses the `on` attribute to install event handlers on elements. Like attributes, some events and actions are available to common elements while others are special to certain components. For our first page, we’ll register a basic event, a user’s click, then respond with a popular action, hide.
+
+Add a button to your page, and give it the `on` attribute:
+
+```html
```

Comment:
For this and other snippets in the intro, I'm worried about people copying code that doesn't actually work due to it being from a partially-correct example. Is the current example of `on="event:targetId.action"` too confusing?

Reply:
I'm not too worried about it being wrong. Because it's a tutorial broken down into steps it isn't intended to work at every step and we talk about the playground throwing validation errors. Maybe we add a link to the finished code at the beginning of the tutorial so if readers get lost/confused they can compare with the end result - wdyt?

Reply:
Sounds good!